scripts/monitoring/auto_start_or_stop_workers.py

```diff
@@ -46,32 +58,54 @@ def get_all_challenges():
     return response.json()
 
 
-def start_or_stop_workers_for_active_challenges(response):
+def is_unapproved_challenge(workers, approved_by_admin, created_at):
+    current_date = utc.localize(datetime.now())
+    return (
+        workers is not None
+        and not approved_by_admin
+        and (current_date - created_at).days >= 3
+    )
+
+
+def start_or_stop_workers_for_challenges(response):
     for challenge in response["results"]:
         challenge_id = challenge["id"]
         workers = challenge["workers"]
+        approved_by_admin = challenge["approved_by_admin"]
         is_docker_based = challenge["is_docker_based"]
         challenge_start_date = parser.parse(challenge["start_date"])
         challenge_end_date = parser.parse(challenge["end_date"])
+        created_at = parser.parse(challenge["created_at"])
         current_date = utc.localize(datetime.now())
 
         print(
-            "Start submission worker for challenge id {}".format(challenge_id)
+            "Start/Stop submission worker for challenge id {}".format(
```

Comment:
Can we please be specific with the message? Also, why are we using `print` statement? Would this be logged on Cloudwatch?

Reply:
This is a independent cron that we run on prod and staging instance. Do we want the logs on cloudwatch?